scalafmt-tests/src/test/resources/rewrite/RedundantParens.stat

```diff
@@ -745,3 +745,19 @@ object Application {
 
   def main(args: Array[String]): Unit = app.run(args)
 }
+<<< constructor issue
+rewrite.rules = [ RedundantParens]
+===
+class Toto() {
+private def myScalaFmtTest(): String = {
+  val test = new Toto
+  ("test")
```

Comment:
i suspect the problem here might be a little different. if you prepend ONLY to the test title, it will print both formatting details, and the rules invoked should be the same.

perhaps because of redundant paren rewrite, the statement start logic missed this case? can you add another test where the expression in parens doesn't start with a literal? a method call?

Reply:
Ach, you are right, it's ` redundant paren rewrite`. Do we update statement starts with that rule?